source/layout/theme.liquid

```diff
@@ -31,7 +31,7 @@
 
     <div class="grid">
 
-      <main class="site__main">
+      <main class="site__main" aria-label="content">
```

Comment:
`role="content"` isn't a thing.  Maybe you're confusing this with `role="main"`?

`<main aria-label="content">` is a nicer way of announcing the main landmark to screen readers. 

without the aria-label, it would be announced as "main"
with the aria-label i provided, it is announced as "main content"

this isn't a _need to have_ but in testing I've done with screen reader users, it's never been not appreciated.